ateam-common-packets/include/stspin.h

```diff
@@ -0,0 +1,141 @@
+/**
+ * @file motor.h
+ * @author Will Stuckey
+ * @brief 
+ * @version 0.1
+ * @date 2022-06-27
+ * 
+ * @copyright Copyright (c) 2022
+ * 
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+// if C11 or newer is used, include assert_static macro for testing
+#if defined( __STDC_VERSION__ ) && __STDC_VERSION__ >= 201112L
+#include <assert.h>
+#endif
```

Comment:
Do we want to enable these assertions in C++? If so, do we need to check `__cplusplus` as well?

Reply:
C++ support added